components/03-modules/range/range.js

```diff
@@ -64,5 +59,4 @@
 
   init();
 
-
 }());
```

Comment:
when i will render in home view like this 
```
{{ render '@range' }}

{{ render '@range' render }}
```
with passing to the second one other config i've got an error with `Error: noUiSlider (10.0.0): Slider was already initialized.`. IMO we should do this script to use multiple times in preview i.e on products site and in filters we can have our size of shoes and price in range.